### PR TITLE
Ordeal Despawns

### PR DESCRIPTION
**Pull request by PotatoTomahto**
## About The Pull Request
[BETACORP CHANGES REVERTED SINCE DERP'S IS BETTER] Lore papers have been fixed. Amber dawn, green dawn, and green noons spawned from dusks now disappear upon death. Greens are now butcherable.

## Why It's Good For The Game
Derp's map changes are better. Janitor clerks and abnorm coders can now rejoice as dusk ordeals no longer clutter entire hallways with bodies.

## Changelog
:cl:
add: green butchering and certain ordeal despawns
add: gives clerk hunting knives (nerfed damage)
tweak: boxes have now explorer masks since they look cooler
fix: lore papers
:cl:

### CODE REVIEW

**Comment by Kitsunemitsu** [comment]
Don't make them despawn, make them butcher.
Give all clerks a butchering knife

**Comment by Kitsunemitsu** [comment]
I want to say that this is made as a stealth nerf in bad faith.
image

This Coder knows that this is a stealth nerf to certain abnormalities and ordeals.

I suggested to just make it a subtype of the Robots that disappear.
Have the factory spawn the subtype and dawn and noon stay.

Rest can stay.

**Review by EgorDinamit — CHANGES_REQUESTED** [review]
Ordeals shouldn't despawn on death, that's what clerks are supposed to be doing.

File: code/modules/mob/living/simple_animal/hostile/ordeal/green.dm [hunk 1]
```diff
@@ -58,7 +60,18 @@
 				H.apply_damage(20, WHITE_DAMAGE, null, H.run_armor_check(null, WHITE_DAMAGE), spread_damage = TRUE)
 			finishing = FALSE
 
-// Green dawn
+//Green dawn factory spawn
+/mob/living/simple_animal/hostile/ordeal/green_bot/factory
```
Comment (EgorDinamit) [inline code/modules/mob/living/simple_animal/hostile/ordeal/green.dm:64]: There is absolutely no reasons to add an entire subtype for it.
Just make factory register a signal for death.

**Pull request merged**